Mlem/App/Views/Root/Tabs/Feeds/SubscriptionListView.swift

```diff
@@ -12,14 +12,15 @@ struct SubscriptionListView: View {
     @Environment(AppState.self) private var appState
     @Environment(NavigationLayer.self) private var navigation
     @Environment(TabReselectTracker.self) var tabReselectTracker
+    @Environment(Palette.self) var palette
     
     @Setting(\.subscriptionSort) private var sort
     
     @State var noDetail: Bool = false
+    @State var feedOptions: [FeedSelection] = FeedSelection.guestCases
```

Comment:
Does this need to be an `@State` value? I reckon it's reasonable enough to use a computed property instead, unless there's a reason we need the `@State`

Reply:
That's a good point--I had it as `@State` because that's how it was in `FeedsView`, but it looks like a computed var works perfectly well in both cases